src/Middleware/OutputCaching/src/IOutputCacheStore.cs

```diff
@@ -1,6 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Buffers;
+using System.IO.Pipelines;
+
 namespace Microsoft.AspNetCore.OutputCaching;
```

Comment:
Will we deprecate/obsolete IOutputCacheStore eventually?

Reply:
No, because the in-process cache store will always prefer to use raw `byte[]` approaches, so will intentionally *not* implement the new API